.readthedocs.yml

```diff
@@ -14,7 +14,8 @@ build:
     post_install:
       - pip install poetry==1.3.0
       - poetry config virtualenvs.create false
-      - poetry install --with doc
+      - poetry install --all-extras
+      - pip install typing-extensions
```

Comment:
Do we wanna move `typing-extensions` into the `pyproject.toml` under the main dependencies group?